### DISCUSSION TITLE
Transmission no longer downloading

### DISCUSSION THREAD

**Discussion by SteveDevDev** [haugene/docker-transmission-openvpn#2046] [General]
I had this working previously, but now downloads are not happening.  There is no error that I can see, but all the torrents say "Downloading" but they never actually download.  They seem to be able to find seeds but never peer with anyone.  I'm using PIA as the VPN provider.

   
docker-compose.yaml

version: "2.1"
services:
  transmission:
    image: haugene/transmission-openvpn
    container_name: transmission
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun
    environment:
      - OPENVPN_PROVIDER=PIA
      - OPENVPN_CONFIG=netherlands,sweden,austria
      - OPENVPN_USERNAME=xxxx
      - OPENVPN_PASSWORD=xxxx
      - LOCAL_NETWORK=172.16.10.0/24
      - TZ=xxxx
      - PUID=1000
      - PGID=1000
    logging:
      driver: json-file
      options:
        max-size: 10m
    volumes:
      - /mnt/media/:/data
      - /etc/localtime:/etc/localtime:ro
    ports:
      - 9091:9091
    restart: unless-stopped

-------------------------------
Logs

Sat Oct 30 22:01:39 2021 /sbin/ip addr add dev tun0 10.14.112.192/24 broadcast 10.14.112.255

Sat Oct 30 22:01:39 2021 /etc/openvpn/tunnelUp.sh tun0 1500 1553 10.14.112.192 255.255.255.0 init

Up script executed with tun0 1500 1553 10.14.112.192 255.255.255.0 init

Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.14.112.192

Updating Transmission settings.json with values from env variables

Using existing settings.json for Transmission /data/transmission-home/settings.json

Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.14.112.192

Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed

Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete

Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091

Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch

sed'ing True to true

Enforcing ownership on transmission config directories

Applying permissions to transmission config directories

Setting owner for transmission paths to 1000:1000

Setting permissions for download and incomplete directories


2

Directories: 775

Files: 664

Setting permission for watch directory (775) and its files (664)

-------------------------------------

Transmission will run as

-------------------------------------

User name:   abc

User uid:    1000

User gid:    1000

-------------------------------------

STARTING TRANSMISSION

Provider PIA has a script for automatic port forwarding. Will run it now.

If you want to disable this, set environment variable DISABLE_PORT_UPDATER=true

Transmission startup script complete.

Sat Oct 30 22:01:40 2021 /sbin/ip route add x.x.x.x/32 via 172.21.0.1

Sat Oct 30 22:01:40 2021 /sbin/ip route add 0.0.0.0/1 via 10.14.112.1

Sat Oct 30 22:01:40 2021 /sbin/ip route add 128.0.0.0/1 via 10.14.112.1

Sat Oct 30 22:01:40 2021 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.

Sat Oct 30 22:01:40 2021 Initialization Sequence Completed

Running functions for token based port fowarding

Reserved Port: 31205  Sat Oct 30 22:01:47 CDT 2021

transmission auth not required

waiting for transmission to become responsive

transmission became responsive

  31    n/a       None  Done         0.0     0.0   None  Will Verify  x

Sum:          356.5 MB               0.0     0.0

setting transmission port to 31205

localhost:9091/transmission/rpc/ responded: "success"

Checking port...

Port is open: Yes

#######################

        SUCCESS        

#######################

Port: 31205

Expiration Sat Jan  1 08:57:52 CST 2022

#######################

Entering infinite while loop

Every 15 minutes, check port status

Reserved Port: 31205  Sat Oct 30 22:16:58 CDT 2021

**Comment by pkishino** [comment]
Change server and try.. nothing wrong in container side

**Reply by SteveDevDev** [reply]
Do you happen to know where I can find a list of the "server names"?  I can't find the list that I should pull from as they are more aliases than FQDNs of the actual server.

**Reply by pkishino** [reply]
https://github.com/haugene/vpn-configs-contrib/blob/main/openvpn/pia/configure-openvpn.sh
thats the script that fetches the PIA ovpn files..you can download them manually and check servers
